src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb

```diff
@@ -3028,86 +3028,15 @@ ProduceBoundNode:
             ' See Section 3 of §11.8.2 Applicable Methods
             ' Deal with Optional arguments. HasDefaultValue is true if the parameter is optional and has a default value.
             Dim defaultConstantValue As ConstantValue = If(param.IsOptional, param.ExplicitDefaultConstantValue(DefaultParametersInProgress), Nothing)
```

Comment:
Since we now don't expect any behavior changes in this file, please revert the file to its original state. The extract method refactorings, if you think they bring a lot of value, could be submitted as a separate PR, but let's not bundle unrelated things in this PR.